gdlsc.go

```diff
@@ -1,104 +1,94 @@
 package main
 
 import (
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"log"
-	"net/http"
 	"os"
-	"os/exec"
+	"path"
+	"path/filepath"
 	"strings"
+
+	"github.com/ryanuber/go-license"
 )
 
-// URL represents path a github repos license
-var URL = "https://api.github.com/repos/%s/license?access_token=" + os.Getenv("ACCESS_TOKEN")
+const root = "vendor/"
 
-//types for message to be printed
-type message struct {
-	Dependency string
-	License    string
-	Link       string
-}
+var files []string
```

Comment:
I think this is fine and not worth spending time on it for such a small package, but in general global variables are a bad practice and should be avoided.